docs/_includes/projects/try/style/after/my-element.js

```diff
@@ -0,0 +1,50 @@
+import { LitElement, html } from '@polymer/lit-element'; 
+
+class MyElement extends LitElement {
+  static get properties(){
+    return {
+      message: { type: String },
+      myArray: { type: Array },
+      myBool: { type: Boolean }
+    };
+  }
+  constructor(){
+    super();
+    this.message='Hello world! From my-element';
+    this.myArray = ['an','array','of','test','data'];
+    this.myBool = true;
+  }
+
+  render(){
+    return html`
+      <!-- Add a style block here -->
+      <style>
+        p {
+          font-family: Roboto;
+          font-size: 24px;
+          font-weight: 500;
+        }
+        .red {
+          color: red;
+        }
+        .blue {
+          color: blue;
+        }
+      </style>
+
+      <p>${this.message}</p>
+
+      <ul>${this.myArray.map(i => html`<li>${i}</li>`)}</ul>
```

Comment:
The use of `i` here could be potentially confusing as `i` usually stands for index. Maybe `it` or `text` would be better here.

Reply:
good point, will edit